<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DLG_NewModel</class>
 <widget class="QDialog" name="DLG_NewModel">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>178</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New Model</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="GBNewModel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="title">
    <string>New Model</string>
   </property>
   <widget class="QPushButton" name="PB_Ok">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>23</y>
      <width>75</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>34</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>23</y>
      <width>311</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="layout_1">
       <item>
        <widget class="QLabel" name="L_Name">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="L_Unit">
         <property name="text">
          <string>Unit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="L_Gravity">
         <property name="text">
          <string>Gravity</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="Layout_2">
       <item>
        <widget class="QLineEdit" name="LE_Name"/>
       </item>
       <item>
        <widget class="QComboBox" name="CB_Unit">
         <item>
          <property name="text">
           <string>MKS(Meter/Kilogram/Newton/Second)</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="CB_GravityDirection">
         <item>
          <property name="text">
           <string>+X</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>+Y</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>+Z</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-X</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-Y</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-Z</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="CBH_SingleFloating">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>123</y>
      <width>211</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Single floating point type for DEM</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="LOpenModel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Model</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Browse">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>167</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-bottom-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 0, 0, 255), stop:0.339795 rgba(255, 0, 0, 255), stop:0.339799 rgba(255, 255, 255, 255), stop:0.662444 rgba(255, 255, 255, 255), stop:0.662469 rgba(0, 0, 255, 255), stop:1 rgba(0, 0, 255, 255));</string>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>LE_Name</tabstop>
  <tabstop>CB_Unit</tabstop>
  <tabstop>CB_GravityDirection</tabstop>
  <tabstop>PB_Ok</tabstop>
  <tabstop>PB_Browse</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
